### PR TITLE
website/integrations: added jellyfin integration

### PR DESCRIPTION
**Pull request by clbx**
# Details

## Changes
* Added Jellyfin Integration to docs.

### CODE REVIEW

File: website/integrations/services/jellyfin/index.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+7. Configure the LDAP Settings as follows:
+   - ``LDAP Server``: ``ldap.company.com``
+   - ``LDAP Port``: 636
+   - ``Secure LDAP``: **Checked**
+   - ``StartTLS``: Unchecked
+   - ``Skip SSL/TLS Verification``: 
+     - If using a certificate issued by a certificate authority Jellyfin trusts, leave this unchecked. 
+     - If you're using a self signed certifcate, check this box.
+   - ``Allow password change``: Unchecked
+     - Since Authentik already has a frontend for password resets, its not nessessary to include this in Jellyfin, especially since it requires bind user to have privileges.
```
Comment (BeryJu) [inline website/integrations/services/jellyfin/index.md:55]: Nit: https://goauthentik.io/developer-docs/docs/writing-documentation#general-guidelines, should be lowercase

File: website/integrations/services/jellyfin/index.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+   - ``Password Reset URL``: Empty
+   - ``LDAP Bind User``: Set this to a the user you want to bind to in Authentik. By default the path will be ``ou=users,dc=company,dc=com`` so the LDAP Bind user will be ``cn=ldap_bind_user,ou=users,dc=company,dc=com``.
```
Comment (BeryJu) [inline website/integrations/services/jellyfin/index.md:57]: Same as above

File: website/integrations/services/jellyfin/index.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+1. Click "Save"
+2. Logout, and login with a LDAP user. Username **must** be used, logging in with email will not work. 
+
+
+
+
+## authentik Configuration
+
+No additional Authentik configuration needs to be configured. Follow the LDAP outpost instructions to create an LDAP outpost and configure access via the outpost
```
Comment (BeryJu) [inline website/integrations/services/jellyfin/index.md:97]: Same as above

**Review by BeryJu — CHANGES_REQUESTED** [review]
Minor nitpicks, needs `make website` to fix linting errors

File: website/integrations/services/jellyfin/index.md (hunk 3 above)
Comment (clbx) [reply website/integrations/services/jellyfin/index.md]: Fixed

File: website/integrations/services/jellyfin/index.md (hunk 2 above)
Comment (clbx) [reply website/integrations/services/jellyfin/index.md]: Fixed

File: website/integrations/services/jellyfin/index.md (hunk 1 above)
Comment (clbx) [reply website/integrations/services/jellyfin/index.md]: Fixed

**Review by clbx — COMMENTED** [review]
style fixed

**Review by BeryJu — COMMENTED** [review]
Looks good, could you run `make website` to fix the small linting issues

**Review by clbx — COMMENTED** [review]
Should be good to go.

**Pull request merged**